src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -169,12 +173,6 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
             ModifierUtils.CheckAccessibility(this.DeclarationModifiers, this, isExplicitInterfaceImplementation, diagnostics, location);
 
             this.CheckModifiers(location, hasBody: true, isAutoPropertyOrExpressionBodied: true, diagnostics: diagnostics);
-
-            if (syntax != null)
```

Comment:
What's the purpose/benefit of moving this check? This seemed a better location (the primary constructor which all others call)

Reply:
ah! so this moved to a much better location where we bind the `=>`, and it means *all* the arrow feature-ability checks are in a *single* location.  Thsi is anti peanut butter smear :)